inst/qml/ConfirmatoryFactorAnalysis.qml

```diff
@@ -163,17 +163,20 @@ Form
 		GroupBox
 		{
 			title: qsTr("Plots")
-			CheckBox { text: qsTr("Misfit plot")     ; name: "misfitplot" }
+			CheckBox { text: qsTr("Misfit plot")     ; name: "misfitPlot" }
```

Comment:
if you already want to rename some of the options according to the style guide, that's ok, but then you will also need to specify the new names in `Upgrades.qml`: https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/jasp-upgrade-qml.md